src/common/components/visualization-toggle.tsx

```diff
@@ -26,7 +26,7 @@ export class VisualizationToggle extends React.Component<VisualizationToggleProp
             className: this.props.className,
             onText: 'On',
             offText: 'Off',
-            ariaLabel: this.props.visualizationName,
+            ariaLabel: this.props.label ? this.props.label : this.props.visualizationName,
```

Comment:
You can simplify this a little bit with a [null coalescing operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing):

```suggestion
            ariaLabel: this.props.label ?? this.props.visualizationName,
```

Reply:
Made the change in the most recent commit